.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question
+    url:  https://github.com/eclipse-theia/theia/discussions
+    about: Please ask and answer questions here.
```

Comment:
is the "answer" part necessary?

Reply:
I wasn't sure what to write so I followed the `about` vscode provided.
We can however update it to anything we'd like.

Would `Please ask questions here` be more suitable?

Reply:
> Would `Please ask questions here` be more suitable?

That's what I was thinking